.github/workflows/psalm.yml

```diff
@@ -0,0 +1,30 @@
+name: psalm
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
```

Comment:
We want to run this on all pushes? 

```suggestion
  push:
  pull_request:
```

Reply:
It's fast, so I don't see why not

Reply:
It was helpful when debugging which is why I did it. But if it makes more sense to make it PR only, that's fine